src/pages/project-wizard/ProjectWizard.tsx

```diff
@@ -220,6 +232,17 @@ class ProjectWizardImpl extends React.Component<Props, State> {
         return await getValidationMessages(this.state.project, currentStep.validationKeys);
     };
 
+    getTitle = (): string => {
+        switch (this.props.action.type) {
+            case "edit":
+                return i18n.t("Edit project");
+            case "clone":
+                return i18n.t("Clone project");
+            default:
```

Comment:
It's better to set `"create"` explicitly, as the switch will not compile if new types are added.